HISTORY.rst

```diff
@@ -3,6 +3,19 @@
 Release History
 ===============
 
+0.11.0
++++++++++++++++
+
+**IoT Central updates**
+
+* Adds preview commands (v1.1-preview).
```

Comment:
Can we itemize the command groups that were added?

Reply:
added